docs-ref-services/service-map.md

```diff
@@ -16,17 +16,25 @@ ms.service: Service Map
 # Azure Service Map packages for Node.js

```

Comment:
Also, let's capitalize `Package` so that is consistent across the docs as this is the title of the document.